spec/defines/key_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 describe 'apt::key', :type => :define do
   let(:facts) { { :lsbdistid => 'Debian' } }
-  GPG_KEY_ID = '4BD6EC30'
+  #GPG_KEY_ID = '4BD6EC30'
```

Comment:
I almost did, but it was late and I'd spent a couple hours rebasing/reformatting :)  I'll remove them.